pkgs/by-name/lm/lm4flash/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  libusb1,
+  pkg-config,
+  lib,
+  writeText,
+}:
+let
+  stellaris-udev-rules = writeText "61.stellpad.rules" ''
+    SUBSYSTEM=="usb", ATTRS{idVendor}=="1cbe", ATTRS{idProduct}=="00fd", MODE="0666"
+  '';
+in
+stdenv.mkDerivation rec {
+  pname = "lm4flash";
+  version = "0.1.3";
+  src = fetchFromGitHub {
+    owner = "utzig";
+    repo = "lm4tools";
+    rev = "v${version}";
+    hash = "sha256-ZjuCH/XjQEgg6KHAvb95/BkAy+C2OdbtBb/i6K30+uo=";
+  };
+  sourceRoot = "${src.name}/lm4flash";
```

Comment:
Have you considered packaging both `lm4flash` and `lmicdiusb` together?
The `Makefile` in the root directory builds both of those.
If you do this, you should change the `pname` to `lm4tools`.

Other distros seem to package both:
https://repology.org/projects/?search=lm4tools

There is one case where it only packages `lm4flash`
https://repology.org/projects/?search=lm4flash

Reply:
I haven't used lmicdiusb before. I tried compiling and running it, but it doesn't seem to work. I checked some lm4tools packages, like [[lm4tools-git](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lm4tools-git)], which only include lm4flash.